Extension/Global/Strings.cs

```diff
@@ -24,6 +24,12 @@ public static class Strings
         public const string SETTINGS_PREFER_SHIELD_TOOLTIP = "Prefer infantry with shields when upgrading to infantry troop classes.";
         public const string SETTINGS_RANGED_PREFERENCE_NAME = "Prefer ranged with specific weapons";
         public const string SETTINGS_RANGED_PREFERENCE_TOOLTIP = "Prefer specific ranged weapons when upgrading to ranged troop classes. (0 for no preference, 1 for Bows, 2 for crossbows)";
+        public const string SETTINGS_BUFFER_DAILY_COST_NAME = "Buffer days to pay daily costs";
+        public const string SETTINGS_BUFFER_DAILY_COST_TOOLTIP = "Number of days to pay daily costs upgrading should not use.";
+        public const string SETTINGS_SPLIT_UPGRADES_NAME = "Split upgrades if decision can't be made";
+        public const string SETTINGS_SPLIT_UPGRADES_TOOLTIP = "Splits upgradepaths when no decision can be made. NOTE: decisions will be made based on culture strengths. More info on the mod's page.";
```

Comment:
Sounds like this is the toggle for the whole feature?

Reply:
Yeah, which reminds me: @tbeswick96 Could you include Information regarding the culturestrengths on the mod's page? Ill type it myself dont worry (or just copy from my mod)